File: srm168/StairClimb.java
import java.lang.*;

public class StairClimb {
	
	public int stridesTaken(int[] flights, int stairsPerStride) {
		float strides = (flights.length - 1) * 2;
		for (int i = 0; i < flights.length; i++) {
			float tmp = flights[i];
			strides += Math.ceil(tmp / stairsPerStride);
			}		
		return (int)strides;
	}
}

File: srm168/stair_climb.py
import math


class StairClimb:
	def stridesTaken(self, flights, stairsPerStride):
		strides = (len(flights) - 1) * 2
		for flight in flights:
			strides += math.ceil(float(flight)/stairsPerStride)
		
		return strides